app/workers/medium_removal_worker.rb

```diff
@@ -1,10 +1,30 @@
+# frozen_string_literal: true
+
 class MediumRemovalWorker
   include Sidekiq::Worker
 
   sidekiq_options queue: :data_low
 
   def perform(medium_src, opts={})
-    MediaStorage.delete_file(medium_src, opts)
-  rescue Aws::S3::Errors::AccessDenied
+    MediaStorage.delete_file(object_store_path(medium_src), opts)
+  rescue Aws::S3::Errors::AccessDenied, Azure::Core::Http::HTTPError
```

Comment:
I'm a bit worried that swallowing this `Azure::Core::Http::HTTPError` right now will silently ignore us attempting to delete the incorrect azure paths and leave us storing (and paying) for file in the az blob store that we don't need.

perhaps changing it back to how it was and let these errors surface in hb for us to check on would be best while we figure out how this is working with the paths post s3 migration? Thoughts? 
```
rescue Aws::S3::Errors::AccessDenied
```

Reply:
Yes, agreed - until we are aware of the cases where this type of error is encountered with Azure, I think its best _not_ to catch the error. I'd rather have a chance to monitor before turning on this error catching.